includes/class-wcmp-frontend-settings.php

```diff
@@ -148,6 +155,20 @@ public function get_checkout_display() {
 			return null;
 		}
 
+		/**
+		 * @param $price
+		 *
+		 * @return string
+		 */
+		public function myparcelbe_options_tax_calculation($price){
```

Comment:
Is dit niet beter?
public function get_total_price_with_tax($price){

Reply:
done